README.md

```diff
@@ -9,21 +9,42 @@ Fleet Telemetry is a server reference implementation for Tesla's telemetry proto
 
 The service handles device connectivity as well as receiving and storing transmitted data. Once configured, devices establish a WebSocket connection to push configurable telemetry records. Fleet Telemetry provides clients with ack, error, or rate limit responses.
 
-## Configuring and running the service
+## Prerequisites
+There are a few system prerequisites/recommendations for installing and deploying a Fleet Telemetry server. It is highly recommended that you run this service in a Debian server. In order for any Fleet Telemetry server to interact with vehicles, you will need to own a website domain and host Fleet Telemetry on that domain, which is defined as the `partner_domain` in the [Fleet API documentation](https://developer.tesla.com/docs/fleet-api#fleet-telemetry).
```

Comment:
For ease of installation the recommendation is to run this service in a Debian server

Reply:
Updated